cmd/ipfs-cluster-ctl/main.go

```diff
@@ -433,18 +433,18 @@ cluster "pin add".
 							formatResponse(c, added, nil)
 						}
 					}
-					if lastBuf[0].added == nil {
+					if lastBuf == nil || lastBuf.added == nil {
```

Comment:
This entire change could be reduced to just `if lastBuf[0] == nil || lastBuf[0].added == nil {`, but I thought dropping the slice made things simpler and more readable.